sdc_scissor/testing_api/test_generator.py

```diff
@@ -14,31 +15,37 @@ def _id_generator():
 
 
 class TestGenerator:
-    def __init__(self, count: int, destination: Path):
+    def __init__(self, count: int, destination: Path,tool: str):
         """
         This class is used to generate tests for a virtual environment.
 
         :param count: Number of tests to be generated
         :param destination: Directory where the tests should be saved
         """
+
         self.count: int = count
         self.__id_generator = _id_generator()
         self.__nr_prefix_digits: int = 5
         self.destination: Path = destination
         self.generated_tests: list[Test] = []
+        self.tool: str=tool
```

Comment:
Please use the same format as the other variables (spaces).

Reply:
fixed